content/docs/authorization/troubleshooting.md

```diff
@@ -157,3 +157,24 @@ kubectl -n <namespace> rollout restart deploy/proxy-server
 kubectl -n <driver-namespace> rollout restart deploy/vxflexos-controller
 kubectl -n <driver-namespace> rollout restart daemonSet/vxflexos-node
 ```
+
+### The Failure of Creating an Authorization RPM
```

Comment:
Can you link to this at the top of the troubleshooting page? 

Minor nitpick, can you use Building instead of Creating?

Reply:
done